src/main/java/org/mamute/auth/LDAPApi.java

```diff
@@ -87,6 +89,7 @@ public void init() {
 			groupAttr = env.get(LDAP_GROUP, "");
 			moderatorGroup = env.get(LDAP_MODERATOR_GROUP, "");
 			lookupAttrs = env.get(LDAP_LOOKUP, "").split(",");
+			useSsl = Boolean.parseBoolean(env.get(LDAP_USE_SSL, "false"));
```

Comment:
Oh yeah, of course - should have deduced that from the surrounding if clause. :-)
Updated my fork, therefore: pull request updated. Looks less clumsy now.